libs/@local/harpc/tower/src/body/size_hint.rs

```diff
@@ -0,0 +1,86 @@
+//! Adapted and vendored from `http-body` crate (https://docs.rs/http-body/latest/src/http_body/size_hint.rs.html)
+
+/// A `Body` size hint
+///
+/// The default implementation returns:
+///
+/// * 0 for `lower`
+/// * `None` for `upper`.
+#[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Default)]
+pub struct SizeHint {
+    lower: u64,
+    upper: Option<u64>,
+}
+
+impl SizeHint {
+    /// Returns a new `SizeHint` with default values
+    #[inline]
+    pub fn new() -> SizeHint {
+        SizeHint::default()
+    }
+
```

Comment:
It's neither adding `const` nor any other behavior, should we remove it and use `default` directly instead?
```suggestion
```

Reply:
SizeHint was vendored in directly from the http-body crate. So didn't want to change things unnecessarily (same with Extensions)

Reply:
Fair enough!